generated_code/aderdg.py

```diff
@@ -174,15 +175,15 @@ def addInit(self, generator):
     generator.add('computeChristoffel', computeChristoffel)
 
   @abstractmethod
-  def addLocal(self, generator):
+  def addLocal(self, generator, platforms):
     pass
 
   @abstractmethod
-  def addNeighbor(self, generator):
+  def addNeighbor(self, generator, platforms):
     pass
 
   @abstractmethod
-  def addTime(self, generator):
+  def addTime(self, generator, platforms):
     pass
```

Comment:
Shouldn't platforms be a class member? In this case we do not need to pass it around every time. Mabye also true for the generator...